MatomoTracker.php

```diff
@@ -248,12 +248,12 @@ public function setApiUrl(string $url)
      * It is recommended to only send UTF-8 data to Matomo.
      * If required though, you can also specify another charset using this function.
      *
-     * @param string $charset
      * @return $this
      */
-    public function setPageCharset($charset = '')
+    public function setPageCharset(string $charset = ''): self
```

Comment:
Typehinting `self` will make it impossible to overwrite a method in an extending class in PHP < 7.4
So would be better to not typehint that until we change the php requirements.

Reply:
@sgiehl agree, good point. Thank you, deleted `: self` everywhere.